openhands/agenthub/codeact_agent/tools/browser.py

```diff
@@ -154,7 +154,12 @@
                         'The Python code that interacts with the browser.\n'
                         + _BROWSER_TOOL_DESCRIPTION
                     ),
-                }
+                },
+                'safety_risk': {
+                    'type': 'string',
+                    'description': "The LLM's assessment of the safety risk of this browser action. This helps the security analyzer determine whether user confirmation is needed.",
```

Comment:
@OpenHands can you maintain a set of rules in a separate file like `tools/security_utils.py`

We need a variable for each tool, like BROWSER_RISK_DESC, BASH_RISK_DESC, etc. Please help me with the refactor